redis/pool.go

```diff
@@ -120,9 +127,11 @@ type Pool struct {
 	// for a connection to be returned to the pool before returning.
 	Wait bool
 
+	// Channel used to synchronize goroutines waiting to get a pool connection.
+	waitCh chan int
```

Comment:
This field is protected by mu.  Move below declaration of mu so that comment on mu applies.